Data/ByteString/Short/Internal.hs

```diff
@@ -216,7 +341,6 @@ indexMaybe :: ShortByteString -> Int -> Maybe Word8
 indexMaybe sbs i
   | i >= 0 && i < length sbs = Just $! unsafeIndex sbs i
   | otherwise                = Nothing
-{-# INLINE indexMaybe #-}
```

Comment:
What's the motivation for removing this and other `INLINE` pragmas?

Reply:
Your comment here: https://github.com/haskell/bytestring/pull/471#discussion_r791311803

Reply:
The link doesn't work for me. I assume the relevant resolved discussion doesn't open up. What did I say?

Reply:
> Yes, `INLINABLE` seems like the better default, because it doesn't increase compile time as much for users.
> 
> Instead of adding a pragma on each function, we could alternatively use the [`-fexpose-all-unfoldings` flag](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/using-optimisation.html#ghc-flag--fexpose-all-unfoldings) in this module. That would reduce visual noise, and eliminate the possibility that we forget to add a pragma.

Reply:
I see. My comment was referring to the pragmas you were _adding_. In this case you're "downgrading" an `INLINE` to an `INLINABLE`. It's not obvious that this is a good idea. @Bodigrim, what do you think?

Reply:
Strictly speaking I agree that this is downgrading, but to be honest I think it’s acceptable  in this case.

Reply:
I suspect that removing the pragmas from `indexMaybe` and `(!?)` is unlikely to cause problems.

But `create` is an important work horse for this module. Unless we get some evidence that removing its `INLINE` pragma does not result in worse code, my preference would be to keep it.